lib/renovate.ts

```diff
@@ -11,8 +11,7 @@ process.on('unhandledRejection', (err) => {
 
 proxy.bootstrap();
 
-// eslint-disable-next-line @typescript-eslint/no-floating-promises
-(async (): Promise<void> => {
+await (async (): Promise<void> => {
```

Comment:
Not sure why this fails, will revert for now
```suggestion
// eslint-disable-next-line @typescript-eslint/no-floating-promises
(async (): Promise<void> => {
```